.rubocop_local.yml

```diff
@@ -0,0 +1,19 @@
+#
+# Overrides
+#
+GlobalVars:
+  AllowedVariables:
+  # Loggers
+  - $log
+  - $vim_log
+#
+# Special Exclusions
+#
+AllCops:
+  TargetRubyVersion: 2.2
```

Comment:
@bdunne unsure of the actual suggestion here.  Will follow your guidance on this.